feature/policy_forwarding/ethomplsogre/otg_tests/encap_gre/README.md

```diff
@@ -0,0 +1,837 @@
+                        "local-label": 361473,
+                        "subinterface": 0
+                      }
+                    }
+                  },
+                  {
+                    "config": {
+                      "endpoint-id": "REMOTE"
+                    },
+                    "endpoint-id": "REMOTE",
+                    "remote": {
+                      "config": {
+                        "remote-label": 361472,
+                        "remote-system": "10.250.15.254",
+                        "virtual-circuit-identifier": 1
+                      }
+                    }
+                  }
+                ]
+              }
+            }
+          ]
+        },
+        "interfaces": {
+          "interface": [
+            {
+              "config": {
+                "id": "customer_interface",
+                "interface": "Bundle-Ether8",
+                "subinterface": 0
+              },
+              "id": "customer_interface"
+            }
+          ]
+        },
+        "name": "GEO_1"
+      },
+```
+
+### Pseudowire configs VLAN mode
+
+```json
+      {
+        "config": {
+          "apply-forwarding-policy": "controlword",
+          "name": "GEO_4",
+          "type": "L2P2P"
+        },
+        "connection-points": {
+          "connection-point": [
+            {
+              "config": {
+                "connection-point-id": "GEO_4"
+              },
+              "connection-point-id": "GEO_4",
+              "endpoints": {
+                "endpoint": [
+                  {
+                    "config": {
+                      "endpoint-id": "LOCAL"
+                    },
+                    "endpoint-id": "LOCAL",
+                    "local": {
+                      "config": {
+                        "interface": "Bundle-Ether9.1709",
+                        "local-label": 361494,
+                        "subinterface": 1709
+                      }
+                    }
+                  },
+                  {
+                    "config": {
+                      "endpoint-id": "REMOTE"
+                    },
+                    "endpoint-id": "REMOTE",
+                    "remote": {
+                      "config": {
+                        "remote-label": 361478,
+                        "remote-system": "10.250.15.250",
+                        "virtual-circuit-identifier": 4
+                      }
+                    }
+                  }
+                ]
+              }
+            }
+          ]
+        },
+        "interfaces": {
+          "interface": [
+            {
+              "config": {
+                "id": "customer_interface",
+                "interface": "Bundle-Ether9.1709",
+                "subinterface": 1709
+              },
+              "id": "customer_interface"
+            }
+          ]
+        },
+        "name": "GEO_4"
+      },
+```
+
+### Tunnels/Next-hop group configs Cisco 
```

Comment:
The README cannot require vendor specific configuration.  A single vendor neutral configuration must be defined.

If a vendor cannot support the vendor neutral configuration, one or more [deviations](https://github.com/openconfig/featureprofiles/blob/main/internal/deviations/README.md) may be proposed in the code (test automation) to implement some alternative configuration.  The [feature owner](https://github.com/openconfig/featureprofiles/edit/main/.github/CODEOWNERS) can review the deviation proposal and decide if it is acceptable.